modules/swagger-codegen/src/main/resources/Java/ApiClient.mustache

```diff
@@ -66,7 +66,7 @@ public class ApiClient {
     objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
     objectMapper.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
     objectMapper.enable(DeserializationFeature.READ_ENUMS_USING_TO_STRING);{{#java8}}
-    objectMapper.registerModule(new JavaTimeModule());{{/java8}}{{^java8}}
+    objectMapper.registerModule(new JSR310Module());{{/java8}}{{^java8}}
```

Comment:
Why did you use JSR310Module ? It's deprecated and the one to use is JavaTimeModule !